app/models/audio_file.rb

```diff
@@ -26,6 +26,7 @@ class AudioFile < BaseModel
   alias_attribute :duration, :length
 
   mount_uploader :file, AudioFileUploader, mount_on: :filename
+  skip_callback :commit, :after, :remove_file! # don't remove s3 file
```

Comment:
or https://github.com/carrierwaveuploader/carrierwave/wiki/How-to%3a-Keep-uploaded-files-%28for-a-specific-uploader%29

not sure which method is better, the `config.remove_previously_stored_files_after_update = false` seems more directly meant for this.